### ISSUE TITLE
Transfer the project to jazzband?

### ISSUE THREAD

**Issue by tfeldmann** [deanmalmgren/textract#498]
[jazzband.co](jazzband.co) (https://github.com/jazzband) helps Open Source projects on GitHub share responsibility for software maintenance.

Textract is a great candidate - it's working, tested and used by many.
It just needs some love in form of dependency updates to the point that many other projects cannot use it anymore.

How do you think about this? Would that be okay for you?

**Comment by traverseda** [comment]
As one of the people brought on to handle this, or ended up getting busy with work and doing a terrible job, I'd be in favor.

**Comment by deanmalmgren** [comment]
This sounds amazing. Yes, please. 

I have not been able to maintain textract for some time now and it's been a source of unnecessary shame. Would love to find a way to give the textract community the support they need.

What are the next steps to transfer ownership?

**Comment by traverseda** [comment]
According to the FAQ you can just transfer it to the jazzband organization: https://jazzband.co/about/guidelines

This should mostly meet the guidelines

**Comment by tfeldmann** [comment]
Sounds great! I opened a proposal ticket to involve the jazzband roadies: https://github.com/jazzband/help/issues/358

**Comment by deanmalmgren** [comment]
Sorry it took me so long to circle back to this. I've had a crazy couple of weeks at work and this thread got buried in my inbox. I believe I filled out the necessary information on the jazzband ticket and I just merged in a PR to support the transfer. I assume that at some point we'll need to add @tfeldmann (or someone else?) as a maintainer of the project to get it moved over to the jazzband org?

**Comment by chenrui333** [comment]
any luck of moving the project to jazzband? Thanks!

**Comment by tfeldmann** [comment]
Nope. No answer yet from the jazzband roadies. Seems like jazzband has its own problems: https://github.com/jazzband/help/issues/196

**Comment by endolith** [comment]
Does that mean jazzband itself needs more volunteers for this to be maintained?

**Comment by KyleKing** [comment]
I'm going to be involved with maintenance moving forward!

Unfortunately it doesn't seem like the Jazzband handoff has panned out, so I'll go ahead and close this issue unless the situation changes
